.github/workflows/ci.yml

```diff
@@ -1,3 +1,5 @@
+name: checks
```

Comment:
I think the list of required workflow statuses needs to be adjusted if the name changes here.

How about something like `Build Status` by the way? I think that always looks nice in the GitHub badges.

Reply:
As we're not only building, but also running the linter and testing, I'd prefer `CI Status` instead of `Build Status`.

Reply:
>  I'd prefer CI Status instead of Build Status

That sounds good :)